scenes/Pilot/Pilot.gd

```diff
@@ -0,0 +1,14 @@
+extends Node2D
```

Comment:
I like this new structure. It's nice to have the flexibility to add new player attributes in the future.